cli/tests/pcluster/cli/test_delete_cluster.py

```diff
@@ -81,6 +81,7 @@ def test_execut_with_wait(self, mocker):
         assert_that(delete_cluster_mock.call_args).is_length(2)  # this is due to the decorator on delete_cluster
         args_expected = {"region": None, "cluster_name": "cluster"}
         delete_cluster_mock.assert_called_with(**args_expected)
+        print(cf_waiter_mock.call_args.kwargs)
```

Comment:
You'd keep the print?

Reply:
No that's a mistake.

Reply:
I was having trouble with previous versions of python that I had to debug with the CI pipeline...

Reply:
(But it should be removed now?)